src/trip.js

```diff
@@ -0,0 +1,14 @@
+class Trip {
+  constructor(trip) {
+    this.id = trip.id;
+    this.userId = trip.userId;
+    this.destinationId = trip.destinationId;
+    this.travelers = trip.travelers;
+    this.date = trip.date;
+    this.duration = trip.duration;
+    this.status = trip.status;
+    this.suggestedActivities = trip.suggestedActivities;
+  }
+}
+
```

Comment:
If you don't end up having any methods on the Trip class, you won't really need to have it since they will be fetched as objects anyway, but it's probably a good idea to have it in case you decide to add methods as you're fleshing out your structure. Something to think about: For your cost method on the destination, how will that class communicate with the Trip class to get the information it needs about duration and number of travelers?